pkgs/development/python-modules/django-cryptography/default.nix

```
@@ -0,0 +1,35 @@
+{ buildPythonPackage, cryptography, django, django-appconf, fetchFromGitHub
+, lib, pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "django-cryptography";
+  version = "1.1";
+  disabled = pythonOlder "3.7";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "georgemarshall";
+    repo = "django-cryptography";
+    rev = version;
```

```suggestion
    rev = "refs/tags/${version}";
```

Prevents branch vs tag confusion.